api-reference/10 UI Components/dxForm/3 Methods/reset(editorsData).md

```diff
@@ -3,10 +3,142 @@ id: dxForm.reset(editorsData)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Resets the editors values to the values passed as an argument. Resets all other values to initial values.
 
 ##### param(editorsData): Object
-<!-- Description goes here -->
+New editors data.
```

Comment:
Again - even if "data" is plural, its modifier/adjective should be singular: "editor data". In other words "editor data" can mean data for multiple editors, no problems with that. But since the description above uses the word "values" instead of "data", I decided to change it a bit. 

```suggestion
New values for editors.
```